model/metrics/go-metrics.yaml

```diff
@@ -0,0 +1,88 @@
+groups:
+  - id: metric.go.memory.used
+    type: metric
+    metric_name: go.memory.used
+    brief: "Virtual memory mapped by the Go runtime."
```

Comment:
Does "mapped" mean committed or reserved?

Reply:
This metric is now no longer virtual memory usage (VSS), and is instead the amount of memory used by the program.  I believe that makes it committed memory.